4_ASP.NET_CORE/task4/NLAYER_server/FilmsCatalog/FilmsCatalog/FilmsCatalog.BLL/ModelsBLL/CommentBLL.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FilmsCatalog.BLL.ModelsBLL
```

Comment:
You could simplify this: 
1. change namespace to "FilmsCatalog.Business.Models"
2. name class as "Comment"